src/shipit_signoff/shipit_signoff/api.py

```diff
@@ -149,32 +155,23 @@ def delete_step(uid):
 
 def is_user_in_group(group):
     '''
-    Dummy function while auth0 wrappers are not in place.
+    Check a user's group membership using auth0
     '''
-    return True
-
-    if auth0.user_loggedin:
-        group_membership = auth0.user_getinfo(['groups']).get('groups')
-        return group in group_membership
-    return False
+    group_membership = auth0.user_getinfo(
```

Comment:
You don't need to check for logged-in-ness here anymore? I guess because all of the endpoints are wrapped in moz_accept_token?